### PR TITLE
Fix on User classes and integration with Database

### PR DESCRIPTION
**Pull request by marusack**

### CODE REVIEW

File: app/src/main/java/com/example/braca/muffinapplication/MainActivity.java [hunk 1]
```diff
@@ -8,60 +8,50 @@
...
 import com.google.firebase.database.DatabaseReference;
 import com.google.firebase.database.FirebaseDatabase;
-import com.google.firebase.storage.StorageReference;
 
-import java.io.File;
 import java.util.HashMap;
 
 import objects.User;
 
+
```
Comment (Braca2) [inline app/src/main/java/com/example/braca/muffinapplication/MainActivity.java]: DO NOT COMMIT BLANK LINES. Regards! :)

File: app/src/main/java/com/example/braca/muffinapplication/MainActivity.java [hunk 2]
```diff
@@ -8,60 +8,50 @@
...
+                String nameString = name.getText().toString().trim();
+                String lastNameString = lastName.getText().toString().trim();
+                String cellphoneString = cellphone.getText().toString().trim();
+                User user = new User(nameString,lastNameString,cellphoneString);
 
+                dbReference.child(user.getId());
+                //1- create a child  in the root object (remember to show the json from firebase)
+                //todo check if this workaround works on
+                HashMap<String, String> datamap = new HashMap<>();
+                datamap.put("FirstName",user.getFirstName());
```
Comment (Braca2) [inline app/src/main/java/com/example/braca/muffinapplication/MainActivity.java]: space after comma is missing.

File: app/src/main/java/objects/User.java [hunk 3]
```diff
@@ -1,32 +1,29 @@
...
 
 public class User {
-    //clase, define comportamiento los atributos (molde a seguir)
-    private UUID userId; //todo find how it indexed each user so the data wont be overwrited
+
+    private String id;
     private String firstName;
     private String lastName;
-    private Integer cellphone; //atributos
+    private String cellphone; //atributos
```
Comment (Braca2) [inline app/src/main/java/objects/User.java]: comment in spanish?

File: app/src/main/java/objects/User.java [hunk 4]
```diff
@@ -1,32 +1,29 @@
...
 
 
     public User(){} //genero entidades unicas ()
```
Comment (Braca2) [inline app/src/main/java/objects/User.java]: comment in spanish?

File: app/src/main/java/com/example/braca/muffinapplication/MainActivity.java [hunk 5]
```diff
@@ -24,18 +24,37 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
 
         dbReference = FirebaseDatabase.getInstance().getReference();
+        //firt setp to get all the element from the view
```
Comment (Braca2) [inline app/src/main/java/com/example/braca/muffinapplication/MainActivity.java]: fix words

File: app/src/main/java/com/example/braca/muffinapplication/MainActivity.java (hunk 1 above)
Comment (marusack) [reply app/src/main/java/com/example/braca/muffinapplication/MainActivity.java]: <3

**Comment by Braca2** [comment]
End PR #6

**Pull request merged**